crates/pallet-bridge-pot-currency-swap/src/tests.rs

```diff
@@ -1,44 +1,135 @@
+use frame_support::sp_runtime::ArithmeticError;
+
 use crate::mock::*;
+use crate::{Balanced, BalancedError};
 
-/// This test verifies that the genesis builder correctly ensures genesis verifier result in case
-/// it returns false.
+/// This test verifies that the genesis builder correctly ensures genesis bridge pot balances
+/// values that are balanced.
 #[test]
-#[should_panic = "invalid genesis bridge pot currency swap related data"]
-fn genesis_verifier_false() {
+fn genesis_balanced_true() {
     with_runtime_lock(|| {
-        let verify_ctx = MockGenesisVerifier::verify_context();
-        verify_ctx.expect().once().return_const(false);
-
         let config = GenesisConfig {
-            swap_bridge_pot: pallet_pot::GenesisConfig {
+            balances_left: pallet_balances::GenesisConfig {
+                balances: vec![
+                    (4201, 10),
+                    (4202, 20),
+                    (4203, 30),
+                    (4204, 40),
+                    (SwapBridgeLeftPot::account_id(), 310),
+                ],
+            },
+            balances_right: pallet_balances::GenesisConfig {
+                balances: vec![
+                    (4211, 20),
+                    (4212, 40),
+                    (4213, 60),
+                    (4214, 80),
+                    (4215, 100),
+                    (SwapBridgeRightPot::account_id(), 120),
+                ],
+            },
+            swap_bridge_left_pot: pallet_pot::GenesisConfig {
+                initial_state: pallet_pot::InitialState::Unchecked,
+            },
+            swap_bridge_right_pot: pallet_pot::GenesisConfig {
                 initial_state: pallet_pot::InitialState::Unchecked,
             },
```

Comment:
Why unchecked? This should be `Initialized`

Reply:
My bad, thanks, fixed - https://github.com/humanode-network/humanode/pull/700/commits/ba986051de0010f441215de9a226ae9c3c9407e9.